packages/zowe-explorer-api/src/vscode/ui/WebView.ts

```diff
@@ -80,18 +128,21 @@ export class WebView {
             title: this.title,
         });
         this.webviewContent = builtHtml;
-        if (onDidReceiveMessage) {
-            this.panel.webview.onDidReceiveMessage(async (message) => onDidReceiveMessage(message));
+        if (this.webviewOpts?.onDidReceiveMessage) {
+            webviewView.webview.onDidReceiveMessage(async (message) => this.webviewOpts.onDidReceiveMessage(message));
         }
-        this.panel.onDidDispose(() => this.dispose(), null, this.disposables);
-        this.panel.webview.html = this.webviewContent;
+        webviewView.onDidDispose(() => this.dispose(), null, this.disposables);
+        webviewView.webview.html = this.webviewContent;
+        this.view = webviewView;
     }
 
     /**
      * Disposes of the webview instance
      */
-    private dispose(): void {
-        this.panel.dispose();
+    protected dispose(): void {
+        if (this.panel) {
+            this.panel.dispose();
+        }
```

Comment:
What do you think of...

```suggestion
        this.panel?.dispose();
```